datahub-web-react/src/app/shared/ManageAccount.tsx

```diff
@@ -19,14 +20,24 @@ const defaultProps = {
 
 export const ManageAccount = ({ urn: _urn, pictureLink: _pictureLink }: Props) => {
     const entityRegistry = useEntityRegistry();
-
+    const themeConfig = useTheme();
     const handleLogout = () => {
         isLoggedInVar(false);
         Cookies.remove(GlobalCfg.CLIENT_AUTH_COOKIE);
     };
-
     const menu = (
         <Menu>
+            {themeConfig.content.menu.map((value) => {
+                return (
+                    <Menu.Item>
```

Comment:
i think this element needs a `key` property:

https://reactjs.org/docs/lists-and-keys.html